frontend/app/src/components/home/MessageEntry.svelte

```diff
@@ -76,6 +76,9 @@
     $: isGroup = chat.kind === "group_chat";
     $: messageIsEmpty = (textContent?.trim() ?? "").length === 0 && fileToAttach === undefined;
 
+    // Update this to force a new textbox instance to be created
+    $: textboxId = Symbol();
```

Comment:
this doesn't need to be reactive since it is not computed. Seems like `let textBoxId = Symbol()` would be fine.

Reply:
Done